content-descriptor.json

```diff
@@ -1 +1,11 @@
\ No newline at end of file
+{
+  "installDate": "0001-01-01T00:00:00Z",
+  "assetId": "REPLACE_THIS_WITH_CI_BUILD_NUM",
+  "releaseNotes": "## Release Notes for version 2.5.0",
```

Comment:
Doesnt really matter as we create it from flat each build according to the yml file